api/Dockerfile

```
@@ -3,10 +3,24 @@ ARG LIBHG_REPO_DEPLOY_KEY
 ENV LANG=C.UTF-8 LC_ALL=C.UTF-8
 WORKDIR /build
 
-# Update and install Fortran compiler and f2py(Fortan to python, included in numpy)
+# Update and install f2py(Fortan to python, included in numpy)
 RUN apt-get update && \
-    apt-get install gfortran git -y && \
-    pip install numpy=="1.23.0"
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
```

```suggestion
    apt-get install -y --no-install-recommends \
```
These does not seem needed